client/packages/system/src/IndicatorsDemographics/api/hooks/document/useDemographicIndicatorInsert.ts

```diff
@@ -10,13 +10,7 @@ export const useDemographicIndicatorInsert = () => {
     queryClient.invalidateQueries(api.keys.baseIndicator());
   const { mutateAsync: insertDemographicIndicator } = useMutation(
     async (demographicIndicator: DemographicIndicatorFragment) =>
-      api.insertIndicator(demographicIndicator),
-    {
-      onSettled: () => queryClient.invalidateQueries(api.keys.baseIndicator()),
```

Comment:
I'm guessing you removed these because any changes are already reflected in UI?

Reply:
the code was running the `invalidateQueries` after all of the save actions - having it in here means that the cache is invalidated after each row is saved, and that's not necessary